src/types/onyx/ExpensifyCardSettings.ts

```diff
@@ -9,7 +9,7 @@ type ExpensifyCardSettings = OnyxCommon.OnyxValueWithOfflineFeedback<{
     remainingLimit: number;
 
     /** The total amount of cash back earned thus far */
-    cashBack: number;
+    earnedCashback: number;
```

Comment:
This should be optional same as `remainingLimit` and `currentBalance`
```suggestion
    earnedCashback?: number;
```